Remove reference creation for namespaced branches

krlvi opened the pull request:
This is from before the time where we were using real branches in git. Now that we do that, these serve no purpose.

Caleb-T-Owens reviewed (changes requested):
You can put the entire for loop in the if statement

krlvi commented:
> You can put the entire for loop in the if statement

Done

Commit:
Remove reference creation for namespaced branches

This is from before the time where we were using real branches in git. Now that we do that, these serve no purpose.
